test_streamy.py

```diff
@@ -2,8 +2,11 @@
 
 
 class StreamyTest(unittest.TestCase):
+    def _stream(self, s):
```

Comment:
Ah, very neat idea!  You run all the tests with one setting, and then with the other!